testapp/src/main/res/values/strings.xml

```diff
@@ -105,6 +105,8 @@
     <string name="error_desc_miniapp_version_mismatch">Please check with the project \nadministrator. See our Help Center for \nmore details.</string>
     <string name="error_desc_promotional_text_missing">Promotional text is not available</string>
     <string name="error_desc_miniapp_too_many_request">"Mini app is requested too many times.</string>
+    <string name="error_send_json_to_host_app_invalid_string">Unable to send the message: please check the string and try again</string>
```

Comment:
Are we using this message ?

Reply:
thanks for the heads up! It was a leftover.